### PR TITLE
History

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Video add to histroy on Play
- clear History 
- remove individual video from history

### CODE REVIEW

**Review by narharikale — APPROVED** [review]
clean code ✨
liked the background images😍.


were-good-to-go-lets-do-this

**Comment by Digambar1411** [comment]
Looks great nice UI

**Review by Digambar1411 — APPROVED** [review]
Looks great. And  clean UI

File: src/Services/historyServices/deleteHistoryService.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  try {
+    const response = await axios.delete(`/api/user/history/all`, {
+      headers: {
+        authorization: token,
+      },
+    });
+
+    return response;
+  } catch (error) {
+    console.log(error);
```
Comment (manojasarsa) [inline src/Services/historyServices/deleteHistoryService.js:13]: You can make use of `console.error` for handling errors

**Review by manojasarsa — APPROVED** [review]
Rest code looks clean.

**Pull request merged**